### PR TITLE
Make gitignore Ignore Gerber Folder; BoM Updates

### PR DESCRIPTION
**Pull request by dptsolutions**
Just two small things:

1. Adds a new rule to the gitignore to ignore the default folder kicad generated gerber files are saved.
2. Bill of Materials updates in the readme:
    1. Adds Mouser Electronics options
    2. Updates the LCSC option for the DIP24 socket to match the style of socket at Digikey/Mouser.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,13 +184,13 @@ optimization.
...
-R1..R4  |   4   | C172965 | 13-MFR-25FTE52-100KCT-ND | 100 kOhm resistors
-SW1     |   1   | C15781  | 2449-KG04ET-ND           | DIP-4 switch
-U1      |   1   | C72120  | ED3024-ND                | DIP24 Socket (optional)
-U1      |   1   | ---     | 1568-1060-ND             | Arduino Pro Micro (ATmega32U4 16MHz, 5V)
+Part    | Count | LCSC #   | Digikey #                | Mouser Electronics # | Comment
+--------|-------|----------|--------------------------|----------------------|------------------------------------------
+CONN1   |   1   | C77835   | 609-5371-ND              | 523-L77SDA15SA4CH4F  | DB15 female connector
+R1..R4  |   4   | C172965  | 13-MFR-25FTE52-100KCT-ND | 603-MFR-25FTE52-100K | 100 kOhm resistors
+SW1     |   1   | C15781   | 2449-KG04ET-ND           | 642-DS04             | DIP-4 switch
+U1      |   1   | C2846071 | ED3024-ND                | 575-11044624         | DIP24 Socket (optional)
```
[the remark this answers is not in the document]
Comment (dptsolutions) [reply README.md:192]: Yes, I did choose round hole. But I did that b/c of the data sheet of the part from Digikey, that I didn't change - http://www.on-shore.com/wp-content/uploads/2015/09/SAXXX0X0.pdf. Am I not reading it correctly? I admit, I'm a total amateur here. But the Digikey drawings really seem to indicate that it's the round hole type?

[the remark this answers is not in the document]
Comment (necroware) [reply README.md:192]: The LCSC part was added by myself and that Digikey was a contribution, which I didn't check properly. That's of course a mistake as well. May be you can find a proper Digikey part too and fix it in this contribution?

[the remark this answers is not in the document]
Comment (dptsolutions) [reply README.md]: Just pushed an update that reverts to your original LCSC part, and updates the Digikey & Mouser parts to be of the same style as the LCSC one.

[the remark this answers is not in the document]
Comment (necroware) [reply README.md]: Thank you very much!

**Pull request merged**